### PR TITLE
Add transferable value to notes

### PR DESCRIPTION
**Pull request by zancas**
This PR completes some of the work necessary to close:

https://github.com/zingolabs/zingolib/issues/595

Please run `cargo doc --open` and peruse the `LightClient` docs!

### CODE REVIEW

File: zingolib/src/lightclient.rs [hunk 1]
```diff
@@ -2153,6 +2000,207 @@ async fn get_recent_median_price_from_gemini() -> Result<f64, PriceFetchError> {
...
+                });
+
+        (
+            unspent_transparent_notes,
+            spent_transparent_notes,
+            pending_transparent_notes,
+        )
+    }
+
+    /// Return a list of notes, if `all_notes` is false, then only return unspent notes
```
Comment (fluidvanadium) [inline zingolib/src/lightclient.rs:2157]: this does not come up in cargo doc

File: zingolib/src/commands.rs [hunk 2]
```diff
@@ -1513,7 +1471,6 @@ pub fn get_commands() -> HashMap<&'static str, Box<dyn Command>> {
         ("save", Box::new(SaveCommand {})),
         ("quit", Box::new(QuitCommand {})),
         ("list", Box::new(TransactionsCommand {})),
-        ("notes", Box::new(NotesCommand {})),
```
Comment (fluidvanadium) [inline zingolib/src/commands.rs:1516]: please mention that this function is removed in a changelog

File: zingolib/src/lightclient.rs (hunk 1 above)
Comment (zancas) [reply zingolib/src/lightclient.rs:2157]: fn moved to test only.

File: zingolib/src/commands.rs (hunk 2 above)
Comment (zancas) [reply zingolib/src/commands.rs:1516]: Mentioned move to test-only.

**Comment by zancas** [comment]
This PR is entirely superceded by #597

**Pull request closed without merging**